lang/lua-src/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Clean up.
+rm -fr lua*
+
```

Comment:
I think it would be useful to add a VARIABLE `VERSION=5.4.6`, to be easy to update in the future.